docs/install.md

```diff
@@ -416,10 +416,9 @@ use of custom tasks in pipelines.
 most stable features to be used. Set it to "alpha" to allow [alpha
 features](#alpha-features) to be used.
 
-- `embedded-status`: set this flag to "full" to enable full embedding of `TaskRun` and `Run` statuses in the
- `PipelineRun` status. Set it to "minimal" to populate the `ChildReferences` field in the `PipelineRun` status with
-  name, kind, and API version information for each `TaskRun` and `Run` in the `PipelineRun` instead. Set it to "both" to
-  do both. For more information, see [Configuring usage of `TaskRun` and `Run` embedded statuses](pipelineruns.md#configuring-usage-of-taskrun-and-run-embedded-statuses).
+- `embedded-status`: set this flag to "minimal" to populate the `ChildReferences` field in the `PipelineRun` status with
+  name, kind, and API version information for each `TaskRun` and `Run` in the `PipelineRun` instead. For more information,
+  see [Configuring usage of `TaskRun` and `Run` embedded statuses](pipelineruns.md#configuring-usage-of-taskrun-and-run-embedded-statuses).
```

Comment:
let's not remove the details about setting it to "full" -- and let's add details about setting it to "both"

Reply:
Thanks Jerop. Updated.